docs/excel/custom-functions-best-practices.md

```diff
@@ -120,9 +120,9 @@ Keep in mind the following best practices when creating custom functions in your
     }
     ```
 
-## Developing for Excel for Windows and Excel Online simultaneously
+## Additional considerations
 
-To develop an add-in which works both on Excel for Windows and Excel Online, you should avoid accessing the Document Object Model (DOM) or support libraries like jQuery which rely on the DOM. This is because Excel Online uses and renders everything in the browser, whereas Excel for Windows uses the JavaScript runtime, which does not support the DOM and libraries which use it. To learn more about this runtime, see [Runtime for Excel custom functions](custom-functions-runtime.md).
+In order to create an add-in that will run on multiple platforms (one of the key tenants of Office Add-ins), you should not access the Document Object Model (DOM) in custom functions or use libraries like jQuery that rely on the DOM. On Excel for Windows, where custom functions use the [JavaScript runtime](custom-functions-runtime.md), custom functions cannot access the DOM.
```

Comment:
Is it worth calling out that this is a key tenant? I am curious why you added this. Is it just for emphasis, or is this something that isn't spelled out in docs at a higher level when it comes to developing add-ins all up?

Reply:
yes, worth calling it out here.